src/forms.css

```diff
@@ -847,14 +798,38 @@ because it seems to be more cross-browser consistent with all HTML5 input types
   cursor: pointer;
   color: var(--default-secondary-interactive-color);
   font-weight: bold;
-  padding: 0 12px;
-  border-radius: 13px;
+  font-size: var(--font-size-s);
```

Comment:
Missing a line-height rule to match this smaller size. It otherwise looks a little funky 

![lineheight](https://user-images.githubusercontent.com/61150/33999359-41553070-e0b8-11e7-93b9-a907b120f9a7.gif)

Reply:
For selects too the value is currently 24

<img width="518" alt="screen shot 2017-12-14 at 10 22 09 am" src="https://user-images.githubusercontent.com/61150/33999526-b9688e90-e0b8-11e7-9f2f-c70cb01b0e0e.png">

Reply:
That's intentional: look at buttons on mapbox.com – i'm suggesting we use normal line height + small text for buttons.

Reply:
Look at buttons on Account — they have have larger text. Gotta coordinate the ideal here.

Reply:
O_o